db/migrate/20230610014755_create_posts.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.integer :author_id
+      t.string :title
+      t.text :text
+      t.timestamps
+      t.integer :comments_counter, default: 0
+      t.integer :likes_counter, default: 0
+    end
+
+    add_foreign_key :posts, :users, column: :author_id
+
+    add_index :posts, :author_id
+  end
```

Comment:
- [OPTIONAL] Great job so far 👏🏻 However I will suggest to utilizing a more concise and Rails-like syntax.

```ruby
  create_table :posts do |t|
    t.string :title
    t.text :text
    t.integer :comments_counter
    t.integer :likes_counter
    t.timestamps
    t.references :author, foreign_key: { to_table: 'users' }
  end
```

- The data type `references :author, foreign_key: { to_table: 'users' }` column will create author_id and a corresponding index  (but you can also add a column and index in separate commands)